guides/source/active_record_querying.md

```diff
@@ -1788,34 +1937,34 @@ a large or often-running query. However, any model method overrides will
 not be available. For example:
 
 ```ruby
-class Client < ApplicationRecord
+class Customer < ApplicationRecord
   def name
-    "I am #{super}"
+    "I am #{first_name}"
```

Comment:
This is specifically referencing method overrides, so we should be using `super` here and renaming the method.